Part2-API-Schemas/openapi.yaml

```diff
@@ -562,7 +562,7 @@ components:
       type: object
     PathItem:
       type: string
-      pattern: "^(([A-Za-z][A-Za-z0-9_]+)|(\\[[0-9]+\\]))((\\.[A-Za-z][A-Za-z0-9_]+)|(\\[[0-9]+\\])){0,}$"
+      pattern: "^([A-Za-z][A-Za-z0-9_]+)((\\.[A-Za-z][A-Za-z0-9_]+)|(\\[[0-9]+\\])){0,}$"
```

Comment:
Seems quite solid to me. Only small issue I see is that it allows for leading zeros in array indicies, e.g. `foo[0000]` or `foo[00000000000000000000000001]` are considered valid. Not sure if this is intentional but I would suggest to not allow this as it can easily be fixed by using `^([A-Za-z][A-Za-z0-9_]+)((\\.[A-Za-z][A-Za-z0-9_]+)|(\\[(0|[1-9][0-9]*)\\])){0,}$` instead